src/modules/Dashboard/Dashboard.types.ts

```diff
@@ -3,4 +3,5 @@ export type LinkType = {
     title: string
     url: string
     id: string
+    favorite: boolean
```

Comment:
It should be called `isFavourite`, and the same in the database

`boolean` props are usually called `isSomething`